nx-X11/programs/Xserver/hw/nxagent/Keystroke.c

```diff
@@ -204,24 +205,24 @@ static Bool read_binding_from_xmlnode(xmlNode *node, struct nxagentSpecialKeystr
     }
     else if (strcmp((char *)attr->name, "key") == 0)
     {
-      if (strcmp((char *)attr->children->content, "0") != 0 && strcmp((char *)attr->children->content, "false") != 0)
-	newkm.keysym = XStringToKeysym((char *)attr->children->content);
-
```

Comment:
I am pretty sure it was me who added that code (copy from some lines below), but it is not neccessary. I had added it to allow for "key=0" in my fruitless attempts to implement modifier-only keystrokes.